frontend/src/components/dashboard/subdomain/ConfirmationForm.module.scss

```diff
@@ -40,21 +40,34 @@
     justify-content: space-between;
     padding-top: $spacing-xl;
 
+    button[type="submit"] {
+      background: $color-blue-50;
+      opacity: 0.4;
+
+      &:not(:disabled) {
+        opacity: 1;
+      }
+
+      &:hover:not(:disabled) {
+        background-color: $color-blue-60;
+        opacity: 1;
```

Comment:
Also this line seems redundant with the one from line 48?

Reply:
This applies a `:hover` style on non-disabled buttons.